lib/Jsrt/JsrtHelper.cpp

```diff
@@ -89,7 +89,7 @@ void JsrtCallbackState::ObjectBeforeCallectCallbackWrapper(JsObjectBeforeCollect
         pthread_key_create(&s_threadLocalDummy, DISPOSE_CHAKRA_CORE_THREAD);
 #endif
 
-#if defined(CHAKRA_STATIC_LIBRARY)
+#if defined(CHAKRA_STATIC_LIBRARY) || !defined(_WIN32)
```

Comment:
Basically it was failing (other host with libChakraCore.so) until I enabled this part. In the case of ch, we have manual PAL_Init... call in wmain. Other hosts only call JSRT. This code was excluded and InitializeProcess() was no-op. PAL_Init... was never called, and `DllMain` was not called either.